File: Course-Registration-System-main/python-(iteration_2)/src/Staff.py
from User import User

class Staff(User):
	
    def __init__(self, id, password, name, surname):
        super().__init__(id, password, name, surname)

    def getInfo(self):
      
        return f"Name: {self.getName()}\nSurname: {self.getSurname()}"

File: Course-Registration-System-main/java-(iteration_1)/src/Staff.java
public class Staff extends User {

    public Staff(int id, String password, String name, String surname) {
        super(id, password, name, surname);

    }

    @Override
    public String getInfo() {
        // Implementation for viewing general staff information
        return("Name: " + getName() + "\nSurname: " + getSurname());
    }

}


